src/app.js

```diff
@@ -98,7 +100,11 @@ app.post('/api/command', (req, res) => {
   respond(req.body)
     .then(() => res.send('Success!'))
     .catch(err => {
-      res.send('Something is wrong, please try again later');
+      if (errorMap.get(err.code)) {
+        res.send(errorMap.get(err.code))
+        winston.error(`${err.code} ${err.message}`)
```

Comment:
logging to Winston should happen regardless of whether we can find the error code or not. The purpose of the error map is for users understand why they should do next